wremnants/CardTool.py

```diff
@@ -813,12 +818,27 @@ def loadPseudodataFakes(self, datagroups, forceNonzero=False):
         return hdata
 
     def loadPseudodata(self, forceNonzero=False):
+
+        hdatas = []
+
+        if self.pseudodataFitInput:
+            channel = self.pseudoDataFitInputChannel
+            for idx, pseudoData in enumerate(self.pseudoData):
+                if pseudoData == "nominal":
+                    phist = self.pseudodataFitInput.nominal_hists[channel]
+                elif pseudoData == "syst":
+                    phist = self.pseudodataFitInput.syst_hists[channel][{"DownUp" : self.pseudodataFitInputDownUp}]
```

Comment:
Should you not access the index of a specific systematic, otherwise you will have an axis with all systematics no?

Reply:
Yes but the selection of systematic happens later via the systidx (and this part of the code is not changed)

https://github.com/WMass/WRemnants/blob/6b4da5b495e1496c6a72eea597ad8f5ce0ef8121/wremnants/HDF5Writer.py#L222

Reply:
Ah ok I see. Would it be possible and easy to change pseudoData to be the name of the systematic (or "nominal") to allow for adding different pseudodata?

Reply:
Maybe, but doing it like this is more coherent with the existing functionality, in the sense that "pseudodata" refers to the name of the histogram.

The limitation on picking indexes from different histograms already exists in the current functionality and should be solved by more general means probably.